.github/workflows/publish.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v3
         with:
-          python-version: ${{ env.PYTHON }}
+          python-version: "3.11"
```

Comment:
What's the motivation for hard-coding this instead of using the environment Python?

Reply:
gotcha, thanks for clearing that up!